tests/setup_driver.py

```diff
@@ -0,0 +1,41 @@
+import os
```

Comment:
@marco-c Can we have a separate library for driver which can be used in test cases as well as in actual crawler ?

Reply:
Definitely, you can put it in `autowebcompat`.